test/ttmlir/Dialect/TTIR/stablehlo_to_ttir_SubtractOp.mlir

```diff
@@ -0,0 +1,11 @@
+// REQUIRES: stablehlo
+// RUN: ttmlir-opt --convert-stablehlo-to-ttir %s | FileCheck %s
+#any_device = #tt.operand_constraint<dram|l1|scalar|tile|any_device|any_device_tile>
+module @jit_eltwise_subtract attributes {} {
+  func.func public @test_subtract(%arg0: tensor<13x21x3xf32>, %arg1: tensor<13x21x3xf32>) -> tensor<13x21x3xf32> {
+    %0 = stablehlo.subtract %arg0, %arg1 : tensor<13x21x3xf32>
+    // CHECK: %[[C:.*]] = tensor.empty[[C:.*]]
+    // CHECK: %[[C:.*]] = "ttir.subtract"[[C:.*]]
+    return %0 : tensor<13x21x3xf32>
+  }
+}
```

Comment:
Great work Usman! Can you please also add test for stablehlo.divide op conversion?

Reply:
Thanks Marko! I added test for divide op as well.